packages/kbn-expandable-flyout/src/components/preview_section.tsx

```diff
@@ -95,27 +102,30 @@ export const PreviewSection: React.FC<PreviewSectionProps> = ({
           opacity: 0.5;
         `}
       />
-      <div
+      <EuiSplitPanel.Outer
         css={css`
+          margin: ${euiTheme.size.xs};
+          height: 99%;
```

Comment:
it's the bottom, it will stick out a little if height is 100%
![image](https://github.com/elastic/kibana/assets/18648970/09dc7cba-f107-4549-9226-8c42ba96e6c6)